isb_lib/sitemaps/sesar_thing_sitemap.py

```diff
@@ -0,0 +1,95 @@
+
+import datetime
+import typing
+from typing import Optional
+
+from sqlmodel import Session
+
+from isb_lib.core import datetimeToSolrStr
+from isb_lib.sitemaps import SitemapIndexEntry, UrlSetEntry, ThingSitemapIndexEntry, SESARThingUrlSetEntry
+from isb_web.sqlmodel_database import things_for_sitemap
+
+MAX_URLS_IN_SITEMAP = 50000
+
```

Comment:
This file was restored from a previously deleted version.  The SESAR prefix was included to avoid confusion.  All other iSB instances just use the export handler now which has a very different mechanism for writing the .jsonl files.